streams/src/test/java/org/apache/kafka/streams/kstream/internals/KStreamKStreamJoinTest.java

```diff
@@ -44,10 +42,8 @@
     private String topic1 = "topic1";
     private String topic2 = "topic2";
 
-    private IntegerSerializer keySerializer = new IntegerSerializer();
-    private StringSerializer valSerializer = new StringSerializer();
-    private IntegerDeserializer keyDeserializer = new IntegerDeserializer();
-    private StringDeserializer valDeserializer = new StringDeserializer();
+    final private SerDe<Integer> keySerde = new SerDes.IntegerSerDe();
```

Comment:
See, this Freudian slip of `keySerde` suggests to me that we should indeed settle on `Serde` instead of `SerDe`. ;-)